junit-jupiter-api/src/main/java/org/junit/jupiter/api/DisplayNameGeneration.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2015-2018 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.jupiter.api;
+
+import static org.apiguardian.api.API.Status.EXPERIMENTAL;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.apiguardian.api.API;
+
+/**
+ * {@code @DisplayNameGeneration} is used to declare a custom display name
+ * generator for the annotated test class.
+ *
+ * @since 5.4
+ * @see DisplayName
+ * @see DisplayNameGenerator
+ */
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@Inherited
+@API(status = EXPERIMENTAL, since = "5.4")
+public @interface DisplayNameGeneration {
+
+	/**
+	 * Custom display name generator.
+	 *
+	 * @return custom display name generator implementation, defaults
+	 *         to {@link DisplayNameGenerator.StandardGenerator}
+	 */
+	Class<? extends DisplayNameGenerator> value() default DisplayNameGenerator.StandardGenerator.class;
```

Comment:
Since there is now a single attribute in this annotation, there should be no `default` for `value`.

Reply:
Sure. Left over from before the refactoring.

Reply:
Force-pushed required changes.